catalog/chart/templates/deployment.yaml

```diff
@@ -26,9 +26,9 @@ spec:
             - load
           env:
             - name: MONGODB_USERNAME
-              value: okteto
+              value: ""
```

Comment:
@jvc5546 we should use username and password on mongodb. I think updating to the latest version should fix this issue.